doc/user_manual_en.md

```diff
@@ -0,0 +1,111 @@
+# qTox User Manual
+## Index
+
+* [Settings](#Settings)
+* [Groupchats](#Groupchats)
+* [Multi Window Mode](#multi-window-mode)
+* [Keyboard Shortcuts](#keyboard-shortcuts)
+
+## Settings
+### General
+
+#### General Settings
+* _Language:_ Changes which language the qTox interface uses.
+* _Autostart:_ If set, qTox will start when you login on your computer.
+* _Light icon:_ If set, qTox will use a different icon, which is easier to read on black backgrounds.
+* _Show system tray icon:_ If set, qTox will show its icon in your system tray.
+    * _Start in tray:_ On start, qTox will only show its tray icon and no window.
+    * _Minimize to tray:_ The minimize button on the top right, will minimize qTox to its tray icon. There won't be a taskbar item.
+
+* _Auto away after (0 to disable):_ After the specified amount of time, qTox will set your Status to "Away". A setting of 0 will never change your status.
+* _Default directory to save files:_ You can set the directory where qTox puts files you recieved here.
+* _Autoaccept files:_ If set, qTox will automatically accept filetransfers and put them in the directory specified above.
+
+#### Chat
+* _Play sound:_ If checked, qTox will play a sound when you get a new message.
+* _Open window:_ If checked, the qTox window will be opened. <!-- TODO: Will a NEW window be opened? What happens in Multi Window mode? -->
+    * _Focus window:_ If checked, the qTox window will additionally be focused.
+* _Show contacts' status changes:_ ???<!-- TODO: what does this even do?-->
+* _Group chats always notify:_ ???<!-- TODO: what does this even do?-->
```

Comment:
It's about sound (& focus?) notification.
Sometimes groupchats are **really** active, and in general, getting a sound notification about a new message every few seconds for a whole day would be *very* annoying. Thus, by default qTox doesn't make sound when getting a new message from groupchat. If users need to be notified about messages from groupchats, then they can tick this option on.